src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php

```diff
@@ -175,13 +175,16 @@ private function generateItems(OrderInterface $order): void
     {
         $numberOfItems = random_int(1, 5);
         $products = $this->productRepository->findAll();
+        $shippingMethods = $this->shippingMethodRepository->findAll();
+        $shippingMethodsAvailable = (bool) count($shippingMethods);
 
         for ($i = 0; $i < $numberOfItems; ++$i) {
             /** @var OrderItemInterface $item */
             $item = $this->orderItemFactory->createNew();
 
             $product = $this->faker->randomElement($products);
             $variant = $this->faker->randomElement($product->getVariants()->toArray());
+            $variant->setShippingRequired($shippingMethodsAvailable);
```

Comment:
Later, we should improve these fixtures and allow for configuration from yaml file

Reply:
Should I improve the issue description and we keep it open or should I create a new issue for making it configurable?